### ISSUE TITLE
Lighting props that can be turned on or off/unlit versions of lighting props

### ISSUE THREAD

**Issue by agentfortyfour** [Bouncyrock/TaleSpire-Beta-AssetRequests#95]
It would be nice to have unlit versions of the lighting props, or alternatively an option to light and unlight the ones available that the players and gm can interact with similar to the doors (request to open door)

this would add dramatic affects to dungeons, and sort of act as a fog of war in some ways, as well as add to the emersion that they are in a dungeon/crypt etc that has not been touched for decades by another soul.  Coming into an already lit room ruins that feel, and its a pain as a gm to do it on the fly.

-- Leave this next line for others to see --
** ADD :thumbsup:[thumbs up reaction] to give this request more attention.**

**Comment by paul-king** [comment]
_Especially_ for things like the forge/fireplaces & campfires

**Comment by Wrexverdi** [comment]
I wonder if you could place a light producing object, and have it substitute different models with different lighting levels to simulate brightness when you adjust the lighting level? I'm picking somewhat random values here for the range. 
Off/Unlit - 0 tile Sphere
Low - 2 tile sphere
Med - 3 tile sphere (defaullt)
Bright - 5 tile sphere
Really? - 8 tile sphere

Have it selectable from a right click menu, like when you resize a mini as the DM.

**Comment by Wrexverdi** [comment]
If you want to get fancy: Maybe have the lights have a radial menu, one option for brightness, and another for color. Have a wheel with ROYGBIV slots to choose from to keep it (relatively) simple. Assuming a model swap, that does mean... 35 models though.

**Comment by Conspirator05h** [comment]
Yes, instead of having a choice of separate choices of a lit sconce and a choice of an unlit sconce, why not 1 sconce that the DM can (similar to doors) just turn off and on.
